src/Native/Runtime/gcenv.h

```diff
@@ -225,23 +225,7 @@ class SyncBlockCache
 {
 public:
     static SyncBlockCache *GetSyncBlockCache() { return &g_sSyncBlockCache; }
-    void GCWeakPtrScan(void *pCallback, uintptr_t pCtx, int dummy)
-    {
-        UNREFERENCED_PARAMETER(pCallback);
-        UNREFERENCED_PARAMETER(pCtx);
-        UNREFERENCED_PARAMETER(dummy);
-    }
-    void GCDone(uint32_t demoting, int max_gen)
-    {
-        UNREFERENCED_PARAMETER(demoting);
-        UNREFERENCED_PARAMETER(max_gen);
-    }
     void VerifySyncTableEntry() {}
```

Comment:
This method is still being used from `gc.cpp` for the time being, in coreclr master this call is gone too